apps/files/lib/Command/ListFiles.php

```diff
@@ -0,0 +1,311 @@
+<?php
+/**
+ * @copyright Copyright (c) 2024, Nextcloud GmbH
+ *
+ * @author Kareem <yemkareems@gmail.com>
+ * @license AGPL-3.0
```

Comment:
We nowadays use `AGPL-3.0-or-later` if you ok with it:

```suggestion
 * @license AGPL-3.0-or-later
```